testing/build_and_run.sh

```diff
@@ -45,7 +45,7 @@ kubectl create -f $EXAMPLESDIR/kubernetes/gateway.yaml
 sleep 15
 kubectl create -f $EXAMPLESDIR/kubernetes/bookinfo.yaml
 echo "Waiting for the services to come online.."
-sleep 60
+sleep 150
```

Comment:
#248 has removed the `testing/build-scripts/build-examples.sh`,as the integration test isn't based on the `examples/` apps nor deployment scripts any more. The examples deployment scripts should go into the `examples/` dir.